[Error] Installing Waybar with Cava module

Every time I try to install Waybar with Cava module support I get the following error:

<details><summary>Click to show error</summary>
<p>

FAILED: subprojects/cava-0.8.4/libcava.so 
cc  -o subprojects/cava-0.8.4/libcava.so subprojects/cava-0.8.4/libcava.so.p/src_cavacore.c.o subprojects/cava-0.8.4/libcava.so.p/src_common.c.o subprojects/cava-0.8.4/libcava.so.p/src_config.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_common.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_fifo.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_shmem.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_common.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_terminal_noncurses.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_raw.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_noritake.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_terminal_ncurses.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_alsa.c.o subprojects/
cava-0.8.4/libcava.so.p/src_input_portaudio.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_pulse.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_sndio.c.o subprojects/cava-0.8.4/libcava.so.p/src_input_pipewire.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_sdl_cava.c.o subprojects/cava-0.8.4/libcava.so.p/src_output_sdl_glsl.c.o -Wl,--as-needed -Wl,--no-undefined -shared -fPIC -Wl,--start-group -Wl,-soname,libcava.so -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -flto=auto -march=native -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -flto=auto -liniparser -lm /usr/lib/libfftw3.so -pthread -lncursesw -lasound -lportaudio /usr/lib/libpulse.so -lpulse-simple -lsndio /usr/lib/libpipewire-0.3.so /usr/lib/libSDL2.so /usr/lib/libGL.so -Wl,--end-group
{standard input}: Assembler messages:
{standard input}:9: Error: file not found: example_files/config
{standard input}:29: Error: file not found: src/output/shaders/bar_spectrum.frag
{standard input}:49: Error: file not found: src/output/shaders/normalized_bars.frag
{standard input}:69: Error: file not found: src/output/shaders/pass_through.vert
make: *** [/tmp/ccc2NpHx.mk:2: /tmp/ccINWSFu.ltrans0.ltrans.o] Error 1
lto-wrapper: fatal error: make exited with return code 2
compilation aborted.
/usr/bin/ld: error: lto-wrapper failed
collect2: error: ld exited with return code 1
[162/173] Compiling C++ object waybar.p/src_modules_hyprland_workspaces.cpp.o
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Interrupt...
 -> build error: waybar-hyprland-cava-git-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
waybar-hyprland-cava-git - exit status 4

</p>
</details> 

I've tried building directly from git, in which case Waybar doesn't find libcava.so (sudo ldconfig doesn't help). I also tried the wayland-git package from the AUR and this error occurs. I have tried other packages from the AUR but nothing changes. Waybar without the Cava module installs and works great.

WM: Hyprland
OS: Arch
Kernel: 6.4.1-1-cachyos-bore-lto

--- a/meson.build
+++ b/meson.build
@@ -346,7 +346,7 @@ if get_option('experimental')
 endif
 
 cava = dependency('cava',
-                  version : '>=0.8.4',
+                  version : '>=0.8.5',
                   required: get_option('cava'),
                   fallback : ['cava', 'cava_dep'],
                   not_found_message: 'cava is not found. Building waybar without cava')

--- a/subprojects/cava.wrap
+++ b/subprojects/cava.wrap
@@ -1,7 +1,7 @@
 [wrap-file]
-directory = cava-0.8.4
-source_url = https://github.com/LukashonakV/cava/archive/0.8.4.tar.gz
-source_filename = cava-0.8.4.tar.gz
-source_hash = 523353f446570277d40b8e1efb84468d70fdec53e1356a555c14bf466557a3ed
+directory = cava-0.8.5
+source_url = https://github.com/LukashonakV/cava/archive/0.8.5.tar.gz
+source_filename = cava-0.8.5.tar.gz
+source_hash = 9ce3df7d374dc83ed0704fe3caef5e00600ce061d85608aad4142d2c59aa4647
 [provide]
 cava = cava_dep

